coverage.ignore.in

```diff
@@ -0,0 +1 @@
+src/vendor/backward.hpp
```

Comment:
Let's go ahead and ignore the entire `vendor` folder in case anything else is added later on
```suggestion
src/vendor/*
```